Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: loading screen

### Description (*)

This PR adds a loading screen that appears when changing scenes.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-1960](https://jira.unity3d.com/browse/MTT-1960) [MTT-1746](https://jira.unity3d.com/browse/MTT-1746)
### Manual testing scenarios

1. Start a game with a host
2. Join with a client
3. Move the client around
4. Disconnect the client
5. Reconnect the client

The host and the client should see the loading screen when starting the game, then when changing scenes to the CharSelect scene and the BossRoom scene. Those loading screens should hide the slight delay before player characters are spawned, during which the additive scene is not loaded in the BossRoom scene, the camera is at its default position, and the UI is at its default state. This should be more evident during step 5, which should hide the camera movement described in [MTT-1746](https://jira.unity3d.com/browse/MTT-1746)

### Questions or comments

### Contribution checklist
